PLEP-0007.rst

```diff
@@ -179,6 +170,18 @@ Top-Level Sub-Packages
 |   focused on providing package development tools.                            |
 +------------------------------------------------------------------------------+
 
+..
+   +------------------------------------------------------------------------------+
+   | **examples**                                                                 |
+   +------------------------------------------------------------------------------+
+   | :directory: ``./plasmapy/examples/``                                         |
+   | :access: ``plasmapy.examples``                                               |
+   | :PLEP:                                                                       |
+   | :scope:                                                                      |
+   |   A collection of routines showing how to use and/or interface with the      |
+   |   various PlasmaPy tools.                                                    |
+   +------------------------------------------------------------------------------+
+
```

Comment:
Oh, you commented this out! Clever, that was mighty confusing :D

Reply:
Yeah, I didn't want to throw it away until we make the final decision.  It'll be re-implemented or deleted before I remove the "draft" on the PR.